packages/tokens/fungible/src/utils/sac_admin/storage.rs

```diff
@@ -0,0 +1,121 @@
+use soroban_sdk::{contracttype, panic_with_error, token::StellarAssetClient, Address, Env};
+
+use crate::FungibleTokenError;
+
+/// Storage key for accessing the SAC address
+#[contracttype]
+pub enum SacDataKey {
+    Sac,
+}
+
+// ################## QUERY STATE ##################
+
+/// Stores the SAC address, typically called from the constructor.
+///
+/// # Arguments
+///
+/// * `e` - Access to Soroban environment.
+/// * `sac` - The address the SAC contract.
+pub fn set_sac_address(e: &Env, sac: &Address) {
```

Comment:
in some of our (and/or OpenZeppelin's contracts') other designs (the most recent one is `ownable`), we don't have a `setter` for the things that are meant to be initialized at constructor. 

I'm not against this, but if we are going to follow having a separate `setter` function, we should do that consistently across our library.

Just a discussion point 👍

Reply:
Provided some reasoning about that in the AC PR and I think you're also in favor for having explicit setters.